data/nlu.yml

```diff
@@ -0,0 +1,94 @@
+version: "2.0"
```

Comment:
Add a header comment on this file, or on the individual intents if necessary, that explains what's going on here. This is a very significant part of the whole project.

Explain to a developer how it fits into the overall picture.